app/scripts/migrations/126.test.ts

```diff
@@ -0,0 +1,35 @@
+import { migrate, version, PPOMController } from './126';
+
+const oldStorage = {
+  meta: { version },
+  data: {
+    PPOMController: {
+      versionInfo: [
+        {
+          filePath: '/test',
+          hashSignature: 'DUMMY_HASH_SIGNATURE',
+        },
+      ],
+      versionFileETag: 'DUMMY_VERSION_FILE_ETAG',
+    },
+  },
+};
+
+describe('migration #126', () => {
+  it('should remove versionFileETag from PPOMController', async () => {
```

Comment:
Do we need a unit test to clarify nothing happens if no PPOM controller state at all?